sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -811,6 +811,19 @@ public void validate() {
     }
 
     @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
```

Comment:
Can you add a test that verifies this actually makes it through to the primitive transform? Check out [`BigQueryIOTest.testTableSourcePrimitiveDisplayData()`](https://github.com/apache/incubator-beam/blob/master/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java#L648) as an example.